Signed-off-by: dank@kegel.com

See http://sourceware.org/ml/binutils/2005-12/msg00270.html
http://sourceware.org/ml/binutils-cvs/2005-12/msg00139.html

STABS hash table size change, backported from the binutils CVS tree.  Here's
the CVS log comment for the original change:

revision 1.25
date: 2005/12/29 10:29:23;  author: nickc;  state: Exp;  lines: +2 -3
(_bfd_link_section_stabs): Use bfd_hash_table_init rather than
bfd_hash_table_init_n(...,251) so that the size of the hash table can be
controlled by the user.

--- binutils/bfd/stabs.c.old	4 May 2005 15:53:39 -0000	1.24
+++ binutils/bfd/stabs.c	29 Dec 2005 10:29:23 -0000	1.25
@@ -194,9 +194,8 @@
 	goto error_return;
       /* Make sure the first byte is zero.  */
       (void) _bfd_stringtab_add (sinfo->strings, "", TRUE, TRUE);
-      if (! bfd_hash_table_init_n (&sinfo->includes,
-				   stab_link_includes_newfunc,
-				   251))
+      if (! bfd_hash_table_init (&sinfo->includes,
+				 stab_link_includes_newfunc))
 	goto error_return;
       sinfo->stabstr = bfd_make_section_anyway (abfd, ".stabstr");
       if (sinfo->stabstr == NULL)
